src/arkode/arkode_sprk.c

```diff
@@ -23,7 +23,7 @@
 #include "arkode/arkode_butcher.h"
 #include "arkode_impl.h"
 
-ARKodeSPRKTable ARKodeSymplecticEuler(void)
+static ARKodeSPRKTable ARKodeSymplecticEuler(void)
```

Comment:
Maybe `arkodeSymplecticEuler` now that it's private?

Reply:
Yes and similarly for the other internal functions